devtron-labs/dashboard
# fix: Cluster > Node group sorting broken

# Description

To update column metadata when `sortingFieldName` is missing from the `COLUMN_META_DATA` of `node group ` column

<img width="1348" alt="Screenshot 2023-03-14 at 2 51 19 PM" src="https://user-images.githubusercontent.com/61836271/224954822-f0b32b71-e194-42c5-b81b-84210c540594.png">


Fixes [#AB2374)](https://dev.azure.com/DevtronLabs/Devtron/_boards/board/t/OSS%20adoption/Stories/?workitem=2374)

## Type of change


# How Has This Been Tested?
 1. Go to the Cluster > with multiple node
 2. Select Column `Node Group`
 3. Click on the sorting
 4. Page should not break
 5. Remove the applied column from local storage then again repeat 3-4 steps


# Checklist:
